advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+The following steps demonstrate Precisely Connect CDC’s Replication functionality with EDB Postgres Advanced Server.
```

Comment:
Maybe re-write this first sentence as:

After you have configured your EDB Postgres Distribution, as stated in the Configuring section, you will be able to then use Precisely CDC's Replication functionality.  For the examples in this guide, the replication functionality is demonstrated using EDB Postgres Advanced Server.

Reply:
Guide updated.